scripts/migration/README.md

```diff
@@ -19,11 +15,25 @@ nvm install 16
 nvm use 16
 ```
 
-You can then run the migration script with `npx @cosmotech/migrate-azure-sample-webapp v3`
+# Supported versions
+
+## Migrate from v2 to v3
+
+`npx @cosmotech/migrate-azure-sample-webapp v3`
+
+## Migrate from v4 to v5
+
+`npx @cosmotech/migrate-azure-sample-webapp v5 [-s path/to/your/solution.yaml] [-w path/to/your/workspace.yaml]`
+
+Optional parameters:
+- `-s/--solution`: provide the path to an existing YAML solution description file to merge its content with the
+  your webapp configuration file src/config/ScenarioParameters.js and write the result in a new YAML file
```

Comment:
Typo: "the" or "your", not both (the same on lines 31-32) :slightly_smiling_face: